examples/requirements.txt

```diff
@@ -1,3 +1,4 @@
 kopf
 kubernetes
```

Comment:
As far as I can see, the only remaining use of `kubernetes` in `examples/` is in `02-children/example.py`, where it could easily be replaced with Pykube as well. Is it on purpose, to show that Kopf can be used with both libraries?

Reply:
Yes. And also to make sure the official client works (all examples are used as the e2e tests).